<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSection</class>
 <widget class="QDialog" name="DialogSection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add Section</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAddSection">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>440</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Section</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pdAddTimeSLot">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add TimeSlot</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableTimeSlots">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>591</width>
     <height>221</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="autoScrollMargin">
    <number>16</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>98</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>37</number>
   </attribute>
   <column>
    <property name="text">
     <string>Day</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Start Hour</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Start Minutes</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>End Hour</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>End Minutes</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pbRemoveTimeSlots">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>370</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove TimeSlots</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Capacity</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="sbCapacity">
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>155</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Course Code</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbCourseCode"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_section">
      <property name="text">
       <string>Section Code</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbSectionNumber"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
